Controller/Download/Attachment.php

```diff
@@ -63,24 +62,23 @@ public function __construct(
         $this->fileHelper = $fileHelper;
     }
 
-
     /**
-     * @return \Magento\Framework\Controller\ResultInterface
+     * @return ResultInterface|ResponseInterface
      */
-    public function execute() : ResultInterface
+    public function execute()
```

Comment:
czy tutaj zasadne/potrzebne jest usunięcie typu zwracanego?

Reply:
This deletion solves issue, when attachement is of type url. For now more complex refactor of this method, allowing usage of return type is not worth, as general refactor of module is neeeded.